sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -502,10 +502,23 @@ def test_plot_partial_dependence_multiclass(pyplot):
     plot_partial_dependence(clf, iris.data, [0, 1],
                             target='setosa',
                             grid_resolution=grid_resolution)
-    fig = pyplot.gcf()
-    axs = fig.get_axes()
-    assert len(axs) == 2
-    assert all(ax.has_data for ax in axs)
+    fig2 = pyplot.gcf()
+    axs2 = fig2.get_axes()
+    assert len(axs2) == 2
+    assert all(ax.has_data for ax in axs2)
+
+    # 0 is setosa
+    assert all(axs[0].lines[0]._y == axs2[0].lines[0]._y)
+    # 1 is not
```

Comment:
Could you change the comment to be a bit more descriptive

```
# check that the pd plot are different for another target class
```

Reply:
Done